media/js/firefox/hello/hello-ftu-minimal.js

```diff
@@ -0,0 +1,201 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function($) {
+    'use strict';
+
+    var HelloFTU = {};
+    var _helloPanelTimeout;
+
+    // User has clicked "Get Started" button in the Hello panel.
+    HelloFTU.getStartedClicked = false;
+
+    // Tour page has opened Hello Panel
+    HelloFTU.helloPanelVisible = false;
+
+    // Used for unit testing purposes only as "document.hidden" is read only.
+    HelloFTU.documentHidden = null;
+
+    /**
+     * Initialize Hello FTU.
+     * Only start the tour if the Hello icon is present in the tool pallet.
+     * Given that this tour should only be accessed via the Hello panel itself,
+     * this should always be "true". However, it's always possible someone may
+     * share the URL directly with someone who has disabled or removed Hello.
+     */
+    HelloFTU.init = function() {
+        // Hello is still referred to as 'loop' internally for legacy reasons.
+        Mozilla.UITour.getConfiguration('availableTargets', function (config) {
+            if (config.targets && $.inArray('loop', config.targets) !== -1) {
+                HelloFTU.showHelloPanel();
+                HelloFTU.bindPageEvents();
+                HelloFTU.bindHelloEvents();
+                HelloFTU.trackTourSource(HelloFTU.getTourSource());
+            }
+        });
+    };
+
+    HelloFTU.showHelloPanel = function() {
+        // Only open the info panel if tab is visible
+        if (HelloFTU.documentHidden || document.hidden) {
+            return;
+        }
+        // Make sure loop icon is available before opening Hello panel (bug 1111828)
+        Mozilla.UITour.getConfiguration('availableTargets', function (config) {
+            if (config.targets && $.inArray('loop', config.targets) !== -1) {
+                Mozilla.UITour.showMenu('loop', function() {
+                    HelloFTU.helloPanelVisible = true;
+                    $(window).one('resize.hello', HelloFTU.handleResize);
```

Comment:
Do we want/need a `$document.one('click')` handler here to allow a user to dismiss the menu by clicking anywhere on the page? Clicking the Hello icon manually allows the user to dismiss by clicking anywhere on the page - asking more in terms of consistency than necessity.